CHANGELOG.md

```diff
@@ -35,6 +38,18 @@
   * Remove `PaymentMethodNoncesCallback`
   * Remove `PaymentMethodNonce#getDescription()` method
   * `BraintreeClient` constructor no longer throws `InvalidArgumentException`
+  * Make protected static member variables `OPTIONS_KEY`, `OPERATION_NAME_KEY` on `PaymentMethod` package-private
+  * Make `PaymentMethod` constructor package-private
+  * Remove `setValidate` method from `PaymentMethod` base class and all subclasses
```

Comment:
Would it make sense to combine these lines:
```
* Add `Card#setShouldValidate()` method
* Add `PayPalAccount#setShouldValidate()` method
* Add `VenmoAccount#setShouldValidate()` method
```
with this one?

For someone trying to understand how this change impacts a merchant integration, it might help to keep all of this together. Something like:

```
* Rename `setValidate` to `setShouldValidate` and move it from `PaymentMethod` base class to `Card`, `PayPalAccount` and `VenmoAccount` subclasses
```

Reply:
Good idea - updated the CHANGELOG with that single entry